ha_ibmdb2i.h

```diff
@@ -61,6 +61,24 @@ OF SUCH DAMAGE.
 #include "db2i_errors.h"
 #include "db2i_sqlStatementStream.h"
 
+//David Lowes - Temporarily exclude mysql/plugin.h and manually define used constants
```

Comment:
Any idea why you needed to do this? 
IBM has been building against 10.3, so curious if something changed between mariadb versions